bin/rmt-cli

```diff
@@ -34,6 +38,11 @@ if File.exist?(RMT::DEFAULT_MIRROR_DIR) && !File.writable?(RMT::DEFAULT_MIRROR_D
   exit RMT::CLI::Error::ERROR_OTHER
 end
 
+if Settings.try(:host_system).blank? && File.exist?(RMT::CREDENTIALS_FILE_LOCATION) && !File.readable?(RMT::CREDENTIALS_FILE_LOCATION)
```

Comment:
Would it make sense to move this into `RMT::Config.set_host_system!`? 
Thinking of this situation: 
- rmt-cli is called as root
- RMT::Config.set_host_system! successfully reads the credentials
- Process::Sys.seteuid changes to '_rmt' user
- `File.readable?`will fail and print warning message, despite all was fine

Reply:
Semantically speaking (as for the reader of the code) it would be weird to call the same thing twice just because you expect two completely different behaviors. Moreover, that the logging is done by the main script instead of the library method makes more sense to me: 1. try to do X with method call; 2. if what I got from the method call is not what I was expecting, let's log an error.

Reply:
Ok, I see it works because of the `Settings.try(:host_system).blank?` part. So the actual check is skipped because 
the host system was set before. The `File.readable?(RMT::CREDENTIALS_FILE_LOCATION)` would fail at this moment, that's why I was confused.